searx/webapp.py

```diff
@@ -853,7 +853,7 @@ def autocompleter():
 
     for answers in ask(raw_text_query):
         for answer in answers:
-            results.append(str(answer['answer']))
+            results.append(answer['answer'])
```

Comment:
After typing, realized this `str` is completely unnecessary, as its calling `str` on a string already.